lib/vladlev.rb

```diff
@@ -1,11 +1,12 @@
 module Vladlev
-  def self.relative_exists?(filename)
-    File.exists?(File.join(File.dirname(__FILE__), filename))
+  def self.relative_existing_path(filename)
+    file_path = File.join(File.dirname(__FILE__), filename)
+    File.exists?(file_path) && file_path
```

Comment:
`relative_existing_path` does return a path if the file exist instead of just a boolean.

````
File.exists?(file_path) && file_path
````
equal
````
if File.exists?(file_path)
  file_path
end
````